File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.py
# 자릿수 더하기

def solution(n):
    answer = 0
    # str로 먼저 바꾸기
    new = str(n)
    
    # new의 길이를 for 문 돌리기
    for i in range(len(new)):
        # 여기서 int로 바꾸기 
        answer += int(new[i])
    return answer

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;

        String str = Integer.toString(n);
        for (char c : str.toCharArray()) {
            // getNumericValue: char 형을 int 로 변환하는 메소드
            answer += Character.getNumericValue(c);
        }
        return answer;
    }
}